bedrock/mozorg/templates/mozorg/contact/contact-base.html

```diff
@@ -28,50 +29,30 @@ <h1 class="page-title"><span>Where is Mozilla?</span></h1>
       </p>
     </header>
 
-    <div class="map">
-      {% block map %}{% endblock %}
-      <small>
-        {% with link_openstreetmap='http://openstreetmap.org/copyright' %}
-          © <a href="{{ link_openstreetmap }}">OpenStreetMap</a> contributors.
-        {% endwith %}
-      </small>
+    <div class="hero-image">
+      {{ resp_img(
+        url='img/contact/osm/spaces/toronto-office-700.jpeg',
+        srcset={
+        'img/contact/osm/spaces/toronto-office-700.jpg': '500w',
```

Comment:
Nit: this should be `700w`?
```suggestion
        'img/contact/osm/spaces/toronto-office-700.jpg': '700w',
```